test/test-applications/integrations/Directory.Build.props

```diff
@@ -2,9 +2,13 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\Common.props" />
```

Comment:
I would put `<ImplicitUsings>enable</ImplicitUsings>` to Common.props.

Reply:
thanks, didn't notice that one 👍